### ISSUE TITLE
[Bug] Content not shown in docs

### ISSUE THREAD

**Issue by louisdeneve** [huggingface/doc-builder#282]
I'm trying to do inference on multiple gpu's and came accross this link: [https://huggingface.co/docs/transformers/main/en/perf_infer_gpu_many](https://huggingface.co/docs/transformers/main/en/perf_infer_gpu_many). As you can see a 'note' is shown but nothing more, is this expected behaviour or should this be updated?
Thanks in advance

**Comment by mishig25** [comment]
I guess so because the source doc file is https://github.com/huggingface/transformers/blob/main/docs/source/en/perf_infer_gpu_many.mdx

cc: @lvwerra

**Comment by lvwerra** [comment]
Yes, this is work in progress. We'll hopefully update it soon with more info.
